README.md

```diff
@@ -30,6 +30,7 @@ The yml file should be in the following format:
 name: Testing book
 identifier: com.ashfurrow.example
 resources: assets # Optional
+cover: cover.jpeg # Optional
```

Comment:
Are there any restrictions on the file formats that can be used for cover images? Is that something we want to check for?